ckanext/qgov/common/anti_csrf.py

```diff
@@ -160,7 +161,7 @@ def read_token_values(token):
 
     return {
         "message": message,
-        "hash": unicode(parts[0]),
+        "hash": parts[0],
```

Comment:
should this be utf-8 decoded?  maybe add a test for utf-16 and to see that it fails instead of crashes if the hash is encoded wrong.

Reply:
At this point it's a string; we `split` it a few lines earlier.